sphinx/source/schemas/updateAckStateDigest_result.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "items": {
+    "maximum": 255,
+    "minimum": 0,
+    "type": "number"
+  },
+  "title": "updateAckStateDigest/result",
```

Comment:
ackStateDigest?

Reply:
I'd rather keep `updateAckStateDigest` as the operation clearly is a `Write` on the KV and still needs to be followed by the `ack` (as per the new documentation page).